api.js

```diff
@@ -7,19 +7,26 @@ const api = axios.create({
 const getArticlesByCreateDate = () => {
   return api.get("articles?sort_by=created_at").then(({ data }) => {
     return data.articles;
+  }).catch((err) => {
```

Comment:
we shouldn't have catch blocks in our request functions as the error should be dealt with inside the actual component it is being used - this catch block is not adding any value as it's just returning the err which already happens by default when the promise rejects